src/assets/release-notes-1.23.0-alpha.4.json

```
@@ -1435,8 +1517,8 @@
   },
   "97415": {
     "commit": "2face135c730282320d7d7c9873e190e483bce6f",
-    "text": "Podresources interface was changed, now it returns only isolated cpus",
-    "markdown": "Podresources interface was changed, now it returns only isolated cpus ([#97415](https://github.com/kubernetes/kubernetes/pull/97415), [@AlexeyPerevalov](https://github.com/AlexeyPerevalov)) [SIG Node and Testing]",
+    "text": "Podresources interface was changed, now it returns only isolated CPUs",
+    "markdown": "Podresources interface was changed, now it returns only isolated CPUs ([#97415](https://github.com/kubernetes/kubernetes/pull/97415), [@AlexeyPerevalov](https://github.com/AlexeyPerevalov))",
```

nit: suggested case to match documentation
reference: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources

```suggestion
    "text": "PodResources interface was changed, now it returns only isolated CPUs",
    "markdown": "PodResources interface was changed, now it returns only isolated CPUs ([#97415](https://github.com/kubernetes/kubernetes/pull/97415), [@AlexeyPerevalov](https://github.com/AlexeyPerevalov))",
```

---

I will be updating this one in https://github.com/kubernetes/sig-release/pull/1752. Please review there and it will be addressed when next time updating website.